thorlcr/activities/xmlwrite/thxmlwrite.cpp

```diff
@@ -45,7 +45,7 @@ class CXmlWriteActivityMaster : public CWriteMasterBase
         StringBuffer rowTag;
         OwnedRoxieString xmlpath(helper->getXmlIteratorPath());
         if (!xmlpath)
-            rowTag.append("Row");
+            rowTag.append(DEFAULTXMLROWTAG);
```

Comment:
Good to use these, but should put them in common place and also use them in hthor + Roxie.